examples/gatekeeper/imperative/config-map.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
consider renaming `config-map.yaml` file to `settings.yaml` to make it easy to read the text (comment above).

Reply:
I don't agree `settings.yaml` is a good name. IMO it's too vague.